config/config_test.go

```diff
@@ -24,22 +24,27 @@ func TestConfig(t *testing.T) {
 				So(cfg.GracefulShutdownTimeout, ShouldEqual, 5*time.Second)
 				So(cfg.HealthCheckInterval, ShouldEqual, 30*time.Second)
 				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 90*time.Second)
-				So(cfg.KafkaAddr, ShouldResemble, []string{"localhost:9092", "localhost:9093", "localhost:9094"})
-				So(cfg.KafkaVersion, ShouldEqual, "1.0.2")
-				So(cfg.KafkaNumWorkers, ShouldEqual, 1)
-				So(cfg.KafkaSecProtocol, ShouldEqual, "")
-				So(cfg.KafkaSecCACerts, ShouldEqual, "")
-				So(cfg.KafkaSecClientCert, ShouldEqual, "")
-				So(cfg.KafkaSecClientKey, ShouldEqual, "")
-				So(cfg.KafkaSecSkipVerify, ShouldBeFalse)
-				So(cfg.PublishedContentGroup, ShouldEqual, "dp-search-data-importer")
-				So(cfg.PublishedContentTopic, ShouldEqual, "search-data-import")
 				So(cfg.BatchSize, ShouldEqual, 500)
 				So(cfg.BatchWaitTime, ShouldEqual, time.Second*5)
 				So(cfg.ElasticSearchAPIURL, ShouldEqual, "http://localhost:11200")
 				So(cfg.AwsRegion, ShouldEqual, "eu-west-1")
 				So(cfg.AwsService, ShouldEqual, "es")
 				So(cfg.SignElasticsearchRequests, ShouldEqual, false)
+				So(cfg.StopConsumingOnUnhealthy, ShouldBeTrue)
+				So(cfg.Kafka.PublishedContentGroup, ShouldEqual, "dp-search-data-importer")
```

Comment:
Same again, order assertions alphabetically - easier to find whether we have an assertion or not for a configured variable